test/service/test_manifest.py

```diff
@@ -1616,48 +1616,49 @@ def test_hash_validity(self):
-    #        https://github.com/DataBiosphere/azul/issues/2414
-
     @mock.patch('azul.service.manifest_service.ManifestService.get_cached_manifest')
-    def test_fetch_manifest(self, get_cached_manifest):
+    def test_manifest(self, get_cached_manifest):
         """
-        Verify the response from the fetch manifest endpoint for all manifest
-        formats with a mocked return value from `get_cached_manifest`.
+        Verify the response from manifest endpoints for all manifest formats
         """
         for format_ in ManifestFormat:
-            with self.subTest(format_=format_):
+            for fetch in True, False:
                 object_key = 'some_object_key'
-                url = furl(url=self.base_url,
-                           path='/manifest/files',
-                           args=dict(catalog=self.catalog,
-                                     format=format_.value,
-                                     filters='{}',
-                                     objectKey=object_key)).url
-                manifest = Manifest(location=url,
+                azul_manifest_url = furl(url=self.base_url,
+                                         path='/manifest/files',
+                                         args=dict(catalog=self.catalog,
+                                                   format=format_.value,
+                                                   filters='{}',
+                                                   objectKey=object_key)).url
+                manifest_url = 'https://url.to.manifest?foo=bar'
+                manifest = Manifest(location=manifest_url,
                                     was_cached=False,
                                     format_=format_,
                                     catalog=self.catalog,
                                     filters={},
                                     object_key=object_key)
                 get_cached_manifest.return_value = None, manifest
-                # Request the fetch manifest endpoint to verify the response
                 request_url = furl(self.base_url,
-                                   path='/fetch/manifest/files',
-                                   args=dict(format=format_.value,
-                                             filters='{}'))
-                response = requests.get(request_url.url)
-                response_json = response.json()
-                expected_json = {
-                    'Status': 302,
-                    'Location': url,
-                }
-                if format_ == ManifestFormat.curl:
-                    expected_json['CommandLine'] = {
-                        'cmd.exe': f'curl.exe --location "{url}" | curl.exe --config -',
-                        'bash': f"curl --location '{url}' | curl --config -"
-                    }
-                self.assertEqual(expected_json, response_json, response.content)
+                                   path='/manifest/files',
```

Comment:
I tried but it isn't possible without …
```suggestion
                                   path=('/fetch' if fetch else '') + '/manifest/files',
```
… so I left it as it was since there've been some discussions with Hannes on this matter. He indicated that the tradeoff for legibility is worth it.

Reply:
I'm not totally sure what your comment means here, because it looks like the change is there.